src/plot_api/plot_api.js

```diff
@@ -347,7 +347,7 @@ exports.plot = function(gd, data, layout, config) {
 
     // draw ticks, titles, and calculate axis scaling (._b, ._m)
     function drawAxes() {
-        return Axes.doTicks(gd, 'redraw');
+        return Axes.doTicks(gd, graphWasEmpty ? '' : 'redraw');
```

Comment:
OK... this is the only caller using `doTicks(gd, '')` right? From the logic below it looks like there may have been one previously (or someone just calling `doTicks(gd)`) but I don't see it now. I'm a little reticent to add new branches inside `doTicks` when what we really need to do is unpack it into more digestible pieces - but that's probably a project for another time refactoring all of `axes.js`.

Reply:
> OK... this is the only caller using doTicks(gd, '') right? From the logic below it looks like there may have been one previously (or someone just calling doTicks(gd)) but I don't see it now.

That's correct.